docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -252,14 +252,14 @@ This export has been removed and should no longer be used.
 ### 1.x to 2.x
 #### Installation
 
-Install the 2.x version of the `@aws-amplify/ui-react` library.
+Install the 2.x version of the `@aws-amplify/ui-react` library and the 5.x version of the `aws-amplify` library.
```

Comment:
Not sure this is the case, did you check what our aws-amplify peerDependency was at this ui-react version?

Reply:
Yeah looks like it's 3 or 4 so removing the version from aws-amplify